keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -369,7 +376,7 @@ type PodAffinityTerm struct {
   // which pods will be taken into consideration for the incoming pod's pod (anti) affinity. 
   // The default value is empty.
   // +optional
-  MatchLabelSelectors []strinMatchLabelSelectorg
+  MatchLabelSelectors []MatchLabelSelector
```

Comment:
I'm still keen on using two slice to represent the semantics, to avoid another "selector" field to confuse with existing `labelSelector`. How about:

```
type PodAffinityTerm struct {
    matchLabelKeys []string
    mismatchLabelKeys []string
```

where `matchLabelKeys` follows the same naming pattern and semantics with existing ones like:

https://github.com/kubernetes/kubernetes/blob/07646db483d0ee8c53e8292fb90b547fccb75eae/staging/src/k8s.io/api/core/v1/types.go#L3709

and `mismatchLabelKeys` is simply an opposite semantics of `matchLabelKeys`.

cc @ahg-g per https://github.com/kubernetes/enhancements/pull/4001#discussion_r1223234955

Reply:
@msau42 what do you think?

Reply:
Having `matchLabelKeys` and `mismatchLabelKeys` sounds good to me to match what's in `podTopologySpread`. We can continue discussing if there's a better field name later.

Reply:
Sounds good to me; @sanposhiho can you please update the KEP? as @msau42 said, we can debate the names in the implementation PR.

Reply:
OK, I'll update the KEP soon.